.clomonitor.yml

```
@@ -1,3 +1,6 @@
+# see https://github.com/cncf/clomonitor/blob/main/docs/checks.md#exemptions
 exemptions:
   - check: artifacthub_badge
     reason: "Artifact Hub doesn't support Java packages"
+  - check: openssf_badge
+    reason: "ETOOMANYBADGES, but the work has been done: https://www.bestpractices.dev/projects/9992"
```

I'm not too familiar with this, but the linked one is contrib, should we link to the instrumentation one instead?
```suggestion
    reason: "ETOOMANYBADGES, but the work has been done: https://www.bestpractices.dev/projects/9977"
```